src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3608,81 +3619,90 @@ export class StudyViewPageStore
 
     @computed
     get filters(): StudyViewFilter {
-        const filters: Partial<StudyViewFilter> = {};
+        if (!this.filtersRemote.isComplete) {
+            return {} as StudyViewFilter;
+        }
+        return this.filtersRemote.result;
+    }
 
-        const clinicalDataFilters = this.clinicalDataFilters;
+    readonly filtersRemote = remoteData<StudyViewFilter>({
+        await: () => [this.selectedDriverTiersMap],
+        invoke: () => {
+            const filters: Partial<StudyViewFilter> = {};
 
-        const genomicDataIntervalFilters = this.genomicDataIntervalFilters;
-        if (genomicDataIntervalFilters.length > 0) {
-            filters.genomicDataFilters = genomicDataIntervalFilters;
-        }
+            const clinicalDataFilters = this.clinicalDataFilters;
```

Comment:
is there a reason to make this local variable?  we don't seem to it for the other filter types.  i see this isn't new but maybe we can clean it up unless there's a reason

Reply:
Fixed in [860b4a8](https://github.com/cBioPortal/cbioportal-frontend/pull/4552/commits/860b4a806eabd2b81d208148dfdcacd5db0b6661)